components/record/MediaThumbnailGrid.vue

```diff
@@ -1,19 +1,29 @@
 <template>
   <section
     data-qa="media thumbnail grid"
+    class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
     <b-img
       v-for="(thumbnail, index) of thumbnails"
       :key="index"
       :src="thumbnail.src"
       :class="{ 'selected' : isSelected(thumbnail) }"
+      :style="{ 'display' : index > 10 && !showAll ? 'none' : 'inline' }"
       :data-about="thumbnail.about"
       :data-qa="`media thumbnail #${index + 1}`"
       thumbnail
       alt=""
+      class="mb-2 mr-2 rounded-0"
       @click="clickThumbnail(thumbnail.about)"
     />
+    <button
```

Comment:
This needs to load 11 more, not all the rest. That may imply that there would be both a "show more" and "show less" button visible at the same time.

See for instance record /9200355/BibliographicResource_3000117729351 with 159 thumbnails.

Reply:
... I thought, but the acceptance criteria are in fact ambiguous.